ghascompliance/__main__.py

```diff
@@ -50,6 +50,8 @@
     "--github-policy-path",
     default=os.path.join(HERE, "defaults", "policy.yml"),
 )
+github_arguments.add_argument("--retry-count", type=int, default=240)
```

Comment:
Would be good to expose via 

https://github.com/advanced-security/policy-as-code/blob/main/action.yml